src/lib/snark_worker/snark_worker.ml

```diff
@@ -24,7 +24,8 @@ module Worker = struct
     module Get_work = struct
       module V2 = struct
         module T = struct
-          type query = unit
+          (* Contains constraint systems digest for compatibility check *)
+          type query = (string * string) list
```

Comment:
This should probably be `Intf.constraint_system_digests_t` for increased clarity and consistency.

Reply:
When I change it, I get the following from the compiler:
```
File "src/lib/snark_worker/snark_worker.ml", line 28, characters 23-55:                                                                
28 |           type query = Intf.constraint_system_digests_t                                                                           
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                           
Error: Unrecognized type constructor for versioned type 
```

I decided it's easier to not maintain consistency in this case and keep things simple.

Reply:
You'd have to make a versioned type to use in the definition.